packages/documentation-site/patternfly-docs/content/layouts/compass/dashboard.tsx

```diff
@@ -0,0 +1,262 @@
+        >
+          <Flex
+            alignItems={{ default: "alignItemsCenter" }}
+            gap={{ default: "gapMd" }}
+          >
+            Aliyah Frazier
+            <Avatar src={imgAvatar} alt="" size="md" />
+          </Flex>
+        </MenuToggle>
+      )}
+    >
+      <DropdownList>{userDropdownItems}</DropdownList>
+    </Dropdown>
+  );
+
+  const northContent = <CompassHeader logo={<RHAutomationsLogo />} nav={navContent} profile={userDropdown} />;
+
+  const mainContent = (
+    <>
+      <CompassHero gradientDark={{ stop1: '#3d2785', stop2: '#1b0d33', stop3: '#000' }} backgroundSrcDark={heroBg}>
+        <Content>
+          <h1>Automation that does more</h1>
+          <p>
+            Red Hat Ansible Automation Platform offers more capabilities,
+            accessibility, and flexibility, so you can bring the power of
+            automation to the teams, tasks, and environments that need it.
+          </p>
+          <ActionList>
+            <ActionListItem>
+              <Button variant="primary">Upgrade today</Button>
+            </ActionListItem>
+            <ActionListItem>
+              <Button variant="secondary">Talk to a Red Hatter</Button>
+            </ActionListItem>
+          </ActionList>
+        </Content>
+      </CompassHero>
+      <CompassContent>
+          <Grid hasGutter style={{ maxHeight: 'none' }}>
+            <GridItem span={12} sm={12} md={6} lg={4} xl={3} rowSpan={4}>
+              <ClusterDetailsCard />
+            </GridItem>
+            <GridItem span={12} sm={12} md={6} lg={4} xl={3} rowSpan={2}>
+              <AnimationsOverviewClusterInventory />
+            </GridItem>
+            <GridItem span={12} sm={12} md={6} lg={4} xl={3} rowSpan={2}>
+              <AnimationsOverviewStorage />
+            </GridItem>
+            <GridItem span={12} sm={12} md={12} lg={8} xl={3} rowSpan={2}>
+              <AnimationsOverviewMemoryUtilization />
+            </GridItem>
+            <GridItem span={12} sm={12} md={12} lg={12} xl={3} rowSpan={2}>
+              <AnimationsOverviewNetworkActivity />
+            </GridItem>
+            <GridItem span={12} sm={12} md={12} lg={8} xl={6} rowSpan={2}>
+              <RecentActivityCard />
+            </GridItem>
+          </Grid>
+      </CompassContent>
+    </>
+  );
+
+  const eastContent = (
+    <CompassPanel isPill>
+      <ActionList isIconList isVertical /* isVertical */>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Help">
+              <Button variant="plain" icon={<OutlinedQuestionCircleIcon />} aria-label="Help" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Add">
+              <Button variant="plain" icon={<OutlinedPlusSquare />} aria-label="Add" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Help">
```

Comment:
Eric will probably ding you for having 2 identical help buttons with no differentiating label/tooltip.